<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilerFlameWidget</class>
 <widget class="QWidget" name="ProfilerFlameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6">
   <item>
    <layout class="QVBoxLayout" name="controlLayout">
     <item>
      <layout class="QHBoxLayout" name="PIDLayout">
       <item>
        <widget class="QLabel" name="PIDLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>PID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PIDComboBox">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="CPULayout">
       <item>
        <widget class="QLabel" name="CPULabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>CPU:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CPUComboBox">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="protocolLayout">
       <item>
        <widget class="QLabel" name="protocolLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Protocol:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="protocolComboBox">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="durationLayout">
       <item>
        <widget class="QLabel" name="durationLabel">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="durationSpinBox">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="flameGraphLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
